src/mini_games/simple_classics.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+
+def guess_the_number() -> None:
+    """
+    This function implements a typical guess the number game
+    it uses a guess loop until the answer is correct
+
+    :return: None
+    """
+    picked_number: int = random.randint(1, 100)
+    guessed_number: int | None = None
+
+    default_error: str = "Your Input was not a valid number"
+
+    while guessed_number != picked_number:
+        guess = input("Give a number between 1-100 or press enter to cancel: ")
+
+        # Allow the user to exit the game
+        if guess is None:
+            print(f"The Number was {picked_number}, cancelling game...")
+            return
+
+        # Invalid Input Handling
+        else:
+            try:
+                guessed_number = int(guess)
+            except ValueError:
+                print(default_error)
```

Comment:
This is way to handle Errors, you can take a look at [this](<https://www.w3schools.com/python/python_try_except.asp>) if you want to